lms/templates/student_account/hinted_login.underscore

```diff
@@ -0,0 +1,24 @@
+<div class="hint-wrapper">
+    <div class="section-title lines">
+        <h2>
+            <span class="text"><%- gettext("Sign in") %></span>
+        </h2>
+    </div>
+
+    <p><%- _.sprintf( gettext("Would you like to sign in using your %(providerName)s credentials?"), { providerName: hintedProvider.name } ) %></p>
+
+    <button class="action action-primary action-update proceed-button button-<%- hintedProvider.id %> hinted-login-<%- hintedProvider.id %>">
+        <div class="icon fa <%- hintedProvider.iconClass %>" aria-hidden="true"></div>
```

Comment:
Hmm, I'm fine with the default icon, but the sandbox version's display of the dummy or other default-styled buttons doesn't look like the first example above (see the following screengrab).

![screen shot 2015-06-25 at 5 45 35 pm](https://cloud.githubusercontent.com/assets/163763/8369550/9f790ef2-1b89-11e5-8041-fb55809f435c.png)

If you can address that discrepancy and move it to look like your first example, I'm cool with it.